### ISSUE TITLE
Intel m3-8100Y complains 'Unsupported Intel CPU detected'

### ISSUE THREAD

**Issue by strawhatboy** [mq1n/Win11SysCheck#5]
I'm running it with a Intel m3-8100Y which is on the list: [windows11supportedintelprocessors](https://docs.microsoft.com/en-us/windows-hardware/design/minimum/supported/windows-11-supported-intel-processors), while the tool says 'Unsupported Intel CPU detected' 😂

Here's the output:
```log
Windows 11 minimum requirement checker tool: 'C:\Users\straw\Downloads\Win11SysCheck.exe' started!
S Mode checking...
        Version: 10.0
        Product type: 101
S Mode check passed!
CPU checking...
        Active processor count: 4
        Processor arch: 9
        Processor count: 4
        Processor: 1 Speed: 1608
        Processor: 2 Speed: 1608
        Processor: 3 Speed: 1608
        Processor: 4 Speed: 1608
        Processor level: 6 revision: 142 9 vendor: GenuineIntel
Unsupported Intel CPU detected!
Press any key to continue . . .
```

**Comment by mq1n** [comment]
Seems interesting, your CPU was especially blocked in Microsoft's PC Health App by revision id but looks currently allowed in website. Maybe it's outdated check, I allowed it for now.

**Comment by strawhatboy** [comment]
> Seems interesting, your CPU was especially blocked in Microsoft's PC Health App by revision id but looks currently allowed in website. Maybe it's outdated check, I allowed it for now.

Actually it cannot pass the Microsoft's PC Health App check, looks like the CPU is not currently ready for the win11 insider preview, but will be ready for win11 in the future.. 😂 [discussion on reddit](https://www.reddit.com/r/SurfaceGo/comments/o7617u/windows_11_on_surface_go_2/)

**Comment by epman100** [comment]
check out [this recent announcement from MS](https://blogs.windows.com/windows-insider/2021/08/27/update-on-windows-11-minimum-system-requirements-and-the-pc-health-check-app/) regarding updated Win11 minimum requirements (and a revised MS PC Health App check app recently released to Windows insiders)

**Comment by mq1n** [comment]
fixed with 07cbd77
